docs/tutorial/quick-start.md

```diff
@@ -34,6 +53,11 @@ The registration token can be requested by calling the [GitHub API](https://docs
 
 ### Set up the tutorial model
 
+Switch to the LXD controller:
+```
+juju switch lxd
```

Comment:
Can you check if this is the default name when bootstrapping lxd? Or maybe mention that it depends on the name you give this controller.

Reply:
This is the name it gets with the Multipass VM but good point I will add some steps that will read the `juju controller` output and get the lxd name from there